migration/src/main/java/org/corfudb/migration/LogFormat1to2.java

```diff
@@ -83,15 +104,37 @@ public boolean accept(File dir, String name) {
         });
 
         for (File file : foundFiles) {
-            migrateDsFile(file.getAbsolutePath());
+            migrateDsFile(file.getAbsolutePath(), oldEndpoint, newEndpoint);
         }
     }
 
-    public static void migrateDsFile(String path) throws IOException {
+    public static void migrateDsFile(String path, String oldEndpoint, String newEndpoint)
+            throws IOException {
         Path srcPath = Paths.get(path);
         Path destPath = Paths.get(path + ".ds");
 
         byte[] bytes = Files.readAllBytes(srcPath);
+
+        if (layoutStateRecoveryFiles.stream().anyMatch(path::contains)) {
```

Comment:
This method processes files based on certian prefixes, if you want to specifically process particular files you can create another method. Also, make sure you match the suffix and not "any" on the full file patch.

Why cant you use replaceall on the json string instead of all of this?

Reply:
I'll make the matching suffix change. 
I considered replaceAll at first but thought this would be a cleaner way. I can change it back.

Reply:
hmm, isn't that just one line instead of iterating the different types?

Also, that would remove the dependency on the runtime. This module already has a proto file, so depending on runtime would also make it depend on the format/proto file ... The two proto files are not the same.

Reply:
Yes, true.
I've made the change.